compiler/visq/visqlib/QErrorComputer.py

```diff
@@ -104,41 +119,54 @@ def run(self):
                 # To prevent this, relaxed PEIR with epsilon(10^(-6)) is used.
                 rPEIR = PEAK_ERROR / (INTERVAL + 0.000001)
 
-                if tensor_name in qerror_map:
-                    qerror_map[tensor_name] += rPEIR
+                if tensor_name in self.qerror_map:
+                    self.qerror_map[tensor_name] += rPEIR
                 else:
-                    qerror_map[tensor_name] = rPEIR
+                    self.qerror_map[tensor_name] = rPEIR
 
-        for tensor_name, acc in qerror_map.items():
-            qerror_map[tensor_name] = acc / self._num_data
+    def get_final_result(self):
+        qerror_map = dict()
+        for tensor_name, acc in self.qerror_map.items():
+            qerror_map[tensor_name] = acc / self._num_processed_data
 
         return qerror_map
 
+    def run(self):
+        self.advance_on(None, None)
```

Comment:
This change would be necessary for https://github.com/Samsung/ONE/pull/10490/files#r1126078047.
 
```suggestion
        self.advance_on(self._fp32_dir, self._fq_dir)
```